### PR TITLE
Cheatsheet HTML: Added article and address tag

### PR DESCRIPTION
**Pull request by mayank7**
Added article and address tag to HTML cheat sheet in Content Sectioning.

IA page: https://duck.co/ia/view/html_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mayank7 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/html.json [hunk 1]
```diff
@@ -65,6 +65,14 @@
             {
                 "key": "<nav>",
                 "val": "Represents a section of a page that links to other pages or to parts within the page"
+            },
+            {
+                "key": "<article>",
+                "val": "Specifies independent, self-contained content"
+            },
+            {
+                "key": "<address>",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/html.json]: Humm... Not sure this should go in the `Content Sectioning` section (`<article>` is fine). How about we move it to `Document Metadata` or `Text Content`? @duckduckgo/instant-answer-maintainers any thoughts? Not sure which is more appropriate.

Comment (mayank7) [inline share/goodie/cheat_sheets/json/html.json]: @GuiltyDolphin You please suggest whether I should make the changes recommended by you or not, accordingly I will do.

Comment (moollaza) [inline share/goodie/cheat_sheets/json/html.json]: My vote is for the `Text Content` section

Comment (mayank7) [inline share/goodie/cheat_sheets/json/html.json]: @moollaza I moved article and address to Text Content section

**Comment by mayank7** [comment]
@moollaza I moved only address to Text Content section.

**Pull request closed without merging**